+nosnoc/+dcs/stewart.m

```diff
@@ -0,0 +1,118 @@
+classdef stewart < nosnoc.dcs.base
+    properties
+        model
+
+        theta
+        theta_sys
+        lambda
+        lambda_sys
+        mu
+        mu_sys
+
+        z_all
+
+        f_x
+        g_Stewart
+
+        sys_idx
+
+        dims
+
+        % TODO some of these should be in base perhaps
```

Comment:
All 
f_x_fun, 
f_q_fun, 
 g_z_fun, g_alg_fun, It is not clear which of the equations is for the user algebraic and which is DCS algebraic (like e' theta = 1)
g_path_fun
g_comp_path_fun
g_terminal_fun
f_q_T_fun
f_lsq_x_fun
f_lsq_u_fun
f_lsq_T_fun

are generic and could be in the base clase. Why not put them there?

Stewart specific are 
        g_switching_fun
        g_Stewart_fun
        lambda00_fun